### PR TITLE
Chips pipeline description

### PR DESCRIPTION
**Pull request by crazyhottommy**
Hi Roshini,

for this PR:
1. I first git pull from the master and then merge the master to the chips branch on my local repo (there is no conflict).
2. I changed to an absolute path for the chips figure.

we still need to finalize the `chips_output_API.json` when we start working on the integration for ATAC-seq data. 

Thanks a lot for your help!

Best,
Tommy

### CODE REVIEW

**Comment by crazyhottommy** [comment]
Hi Roshini, 
I noticed that the Travis CI build failed, I am not sure what's going on. If you can direct me I can fix on my side.

Thanks!

File: chips/chips_output_API.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            "short_description": "bigwig file",
+            "long_description": "RPKM (reads per kilobase per million) normalized pile up bigwig file for visualization in IGV",
+            "file_purpose": "Analysis view"
+        },
+        {
+            "filter_group": "metrics/quality_metrics_summary",
+            "file_path_template": "analysis/metrics/{run id}.{replicate}/{run id}.{replicate}_metrics_summary.json",
+            "short_description": "quality control summary file",
+            "long_description": "quality control summary file including mapping statistics, PBC, Frip, conservation, contamination statistics",
+            "file_purpose": "Quality Control Metrics"
```
Comment (roshni-b) [inline chips/chips_output_API.json:36]:
```suggestion
            "file_purpose": "Miscellaneous"
```

**Review by roshni-b — APPROVED** [review]
At the moment, permissible values for the `file_purpose` field are: Source view, Analysis view, Clinical view, & Miscellaneous; that's the reason the build failed.

**Comment by roshni-b** [comment]
Build is successful, thank you Tommy!

**Comment by crazyhottommy** [comment]
Thanks for fixing it. Great to hear builds passed! thanks to you and the software team!

**Review by scvannost — APPROVED** [review]

**Pull request merged**